pouchdb-core/index.d.ts

```diff
@@ -426,6 +431,10 @@ declare namespace PouchDB {
         on(event: 'created', listener: (dbName: string) => any): this;
         on(event: 'destroyed', listener: (dbName: string) => any): this;
 
+        debug: debug.IDebug;
+
+        adapter(name: string, adapter: any): void;
```

Comment:
Yeah I can make it match the source.
@nolanlawson, Is Pouchdb.adapter(...) a public API or something 'special'?